src/ontology/hp-edit.owl

```diff
@@ -211572,16 +211611,16 @@ AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_4000079>
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_4000079> "Sensory seeking")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_4000079> <http://purl.obolibrary.org/obo/HP_5200046>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_4000080> (Reduced social initiation)
+# Class: <http://purl.obolibrary.org/obo/HP_4000080> (Reduced initiation of social interaction)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:18064552") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_4000080> "Failure of a person to initiate social interactions with others. For a social interaction to have been initiated by someone, there must have been an absence of social interactions from either party for a period. The interaction must also be unprompted by another party.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_4000080> "Failure of a person to initiate social interactions with others appropriate to culture and developmental level. For a social interaction to have been initiated by someone, there must have been an absence of social interactions from either party for a period. The interaction must also be unprompted by another party.")
 AnnotationAssertion(dc:date <http://purl.obolibrary.org/obo/HP_4000080> "2022-03-02T16:47:44Z"^^xsd:dateTime)
 AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0001-9112-5148>)
 AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0002-4142-7153>)
 AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0002-6670-9157>)
 AnnotationAssertion(dcterms:contributor <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0002-9593-5508>)
 AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_4000080> <https://orcid.org/0000-0002-0736-9199>)
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_4000080> "Reduced social initiation")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_4000080> "Reduced initiation of social interaction")
```

Comment:
@matentzn Didn't we remove this?

Reply:
HP:4000080 is an alternative ID...but the label has been changed again. The narrow synonym is 'Reduced social initiation' because you merged the two terms. BUT you need to retain the original label...at least as a synonym. I think it was 'Abnormal social initiation'? Could you make that a synonym?

Reply:
Also, is it failing because there is an apostrophe in the title? I am not sure that it accepts those

Reply:
Why are you looking at this closed old PR :P 

We have a new one: #9816! Check that!